reddigg.el

```diff
@@ -235,7 +265,7 @@ after deleting the current line which should be the More button."
 
          (insert my-level " " (ht-get data "author") "\n")
          (setq begin (point))
-         (insert (ht-get data "body") "\n")
+         (insert (reddigg--print-text (ht-get data "body")) "\n")
          (setq end (point))
          (reddigg--sanitize-range begin end)
```

Comment:
Hey sorry for the delay, I forgot your MR. I have this function to manually clean up the text. When the  pandoc conversion is on, is this one still necessary?

Reply:
No problem!
Good point, this is not needed after the conversion. Maybe I can move the sanitation process inside the `reddigg--print-text` function and conditionally apply it. This way we can also get rid of for the need of `begin` and `end` variables.